views/standard/spec/indicators.js

```diff
@@ -185,6 +185,7 @@ var signatureStatus = function(pull, node, type, required, signatures) {
       }
    };
 
+<<<<<<< HEAD
```

Comment:
I think you missed a piece

Reply:
Oops, that explains the CI build failure